### PR TITLE
Support Symfony 5

### PR DESCRIPTION
**Pull request by vincentchalamon**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | #54
| License       | MIT

### CODE REVIEW

File: Event/PolyfillEvent.php [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace CoopTilleuls\ForgotPasswordBundle\Event;
+
+use Symfony\Component\EventDispatcher\Event;
+use Symfony\Contracts\EventDispatcher\Event as ContractsEvent;
+
+if (\class_exists(ContractsEvent::class)) {
```
Comment (chalasr) [inline Event/PolyfillEvent.php]: The existence of `ContractsEvent` does not mean that you are using Symfony 4.3+ , but only that `symfony/contracts` is installed. 
See https://github.com/lexik/LexikJWTAuthenticationBundle/pull/644 for the right check :)

File: Manager/ForgotPasswordManager.php [hunk 2]
```diff
@@ -60,10 +63,11 @@ public function resetPassword($propertyName, $value)
         }
 
         // Generate password token
-        $this->dispatcher->dispatch(
-            ForgotPasswordEvent::CREATE_TOKEN,
-            new ForgotPasswordEvent($token)
-        );
+        if (class_exists(Event::class)) {
```
Comment (chalasr) [inline Manager/ForgotPasswordManager.php]: Last one 
```diff
- if (class_exists(Event::class)) {
+ if ($this->dispatcher instanceof ContractsEventDispatcherInterface) {
```

File: Manager/ForgotPasswordManager.php [hunk 3]
```diff
@@ -74,10 +78,11 @@ public function resetPassword($propertyName, $value)
     public function updatePassword(AbstractPasswordToken $passwordToken, $password)
     {
         // Update user password
-        $this->dispatcher->dispatch(
-            ForgotPasswordEvent::UPDATE_PASSWORD,
-            new ForgotPasswordEvent($passwordToken, $password)
-        );
+        if (class_exists(Event::class)) {
```
Comment (chalasr) [inline Manager/ForgotPasswordManager.php]: same here

File: Manager/ForgotPasswordManager.php (hunk 2 above)
Comment (vincentchalamon) [reply Manager/ForgotPasswordManager.php]: Thanks for your review :) It has been fixed

**Pull request merged**